### ISSUE TITLE
Miss Requirements:pyrirgen

### ISSUE THREAD

**Issue by zx1292982431** [jsalt2020-asrdiar/jsalt2020_simulate#17]
required package `pyrirgen` has missed, the repository seems to have been deleted or hidden, how to fix it?

**Comment by HerbHeZhao** [comment]
I have the same question. 
Do you fix it?

**Comment by woshizhishixuebao** [comment]
Hello
Do you fix it?

**Comment by woshizhishixuebao** [comment]
Lol
I saw that author have Removed pyrirgen from environment.yml.

**Comment by TCord** [comment]
Yes, the reference repository `pyrirgen` has been deleted. For an easy fix you can use a fork of it, e.g. [https://github.com/boeddeker/rirgen](https://github.com/boeddeker/rirgen), and change the import from `pyrirgen` to `rirgen`.  
In theory that should keep the simulated data identical to the original version, but I can't be fully sure. Therefore,  I don't think replacing the import would be the best way since it's complicated to be sure that the behavior stays identical.

If you are not interested in obtaining the exact same training data, using `pyroomacoustics` should also work.
